### PR TITLE
[bn] Localize content/bn/api-gateway.md

### PR DESCRIPTION
**Pull request by Saber382**
Signed-off-by: Saber Imtiaz <saberimtiazmitul382@gmail.com>
Localizes `content/bn/api-gateway.md` to Bengali
This is an intial PR as may need some improvements!
Tracking issue: #410

### CODE REVIEW

File: content/bn/api-gateway.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+title: API গেটওয়ে
```
Comment (mitul3737) [inline content/bn/api-gateway.md:2]: Kindly update the title using the title in English. For example, API গেটওয়ে(API Gateway)

**Review by mitul3737 — CHANGES_REQUESTED** [review]

File: content/bn/api-gateway.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+সমস্ত অ্যাক্সেস পরিচালনা এবং নিয়ন্ত্রণ করতে আপনি একটি এন্ট্রি পয়েন্ট চাইবেন।
+অতিরিক্তভাবে, যদি আপনাকে সেই মিথস্ক্রিয়াগুলিতে কার্যকারিতা প্রয়োগ করতে হয়,
+একটি এপিআই গেটওয়ে আপনাকে কোনো অ্যাপ কোড পরিবর্তনের প্রয়োজন ছাড়াই সমস্ত ট্রাফিকের ক্ষেত্রে এটিকে সমানভাবে প্রয়োগ করতে দেয়।
+
+## এটা কিভাবে সাহায্য করে
+
+একটি অ্যাপ্লিকেশনে বিভিন্ন API-এর জন্য একটি একক অ্যাক্সেস পয়েন্ট প্রদান করা,
+API গেটওয়েগুলি সংস্থাগুলির জন্য একটি কেন্দ্রীয় অবস্থানে ক্রস-কাটিং ব্যবসা বা সুরক্ষা যুক্তি প্রয়োগ করে সহজ করে তোলে।
+ এটা অ্যাপ্লিকেশন গ্রাহকদের তাদের সমস্ত প্রয়োজনের জন্য একটি একক ঠিকানায় যাওয়ার অনুমতি দেয়।
+একটি এপিআই গেটওয়ে নিরাপত্তা এবং [observability](/observability/)  এর মতো অপারেশনাল উদ্বেগগুলিকে সহজ করতে পারে ।
```
Comment (mitul3737) [inline content/bn/api-gateway.md:29]: You can update the link for the Bengali page as `bn/observability/`

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by Imtiaz1234 — APPROVED** [review]
LGTM

**Review by mitul3737 — APPROVED** [review]
LGTM

**Pull request merged**